### PR TITLE
Adds Vehicle support and images for views

### PR DESCRIPTION
**Pull request by niamhdurfee**
Fixes issues #51 and #7

### CODE REVIEW

File: src/api/starship.js [hunk 1]
```diff
@@ -26,3 +27,33 @@ starship.fetch = id => {
     }, reject);
   });
 };
+starship.getPicture = searchWord => {
```
Comment (alexkramer) [inline src/api/starship.js]: This picture search appears to be the exact same for each of the component API files. Would prefer moving this to a picture API file and delegating to that.

File: src/components/ViewStarship.vue [hunk 2]
```diff
@@ -47,15 +54,46 @@ export default {
...
+          }
+        });
       });
     },
   },
 };
 </script>
+
+<!-- Add "scoped" attribute to limit CSS to this component only -->
+<style scoped>
```
Comment (alexkramer) [inline src/components/ViewStarship.vue]: This block is copied across the components. Would be a good candidate to push to a style file that can be imported into components that need it.

**Comment by alexkramer** [comment]
Looks pretty good! If possible see if you can add a small test of the new functionality you tested. I also realize that the tests are failing across the builds right now due to Firefox. Do not worry about that, I will take a look and see if I can fix that.

**Comment by niamhdurfee** [comment]
Still looking into writing tests for functionality for #7, fixed issues from before for #51.
The tests are failing on Chrome for me, but FireFox works fine?!

**Pull request closed without merging**